web/packages/teleport/src/mocks/handlers.ts

```diff
@@ -16,30 +16,28 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { rest } from 'msw';
+import { http, HttpResponse } from 'msw';
 
 import cfg from 'teleport/config';
 import { INTERNAL_RESOURCE_ID_LABEL_KEY } from 'teleport/services/joinToken';
 
 // handlersTeleport defines default positive (200) response values.
 export const handlersTeleport = [
```

Comment:
These handlers don't seem to be used anywhere. Grepping the project for `handlersTeleport` returns only this file.